CONTRIBUTING.md

```diff
@@ -74,15 +74,17 @@ Status labels attempt to capture the lifecycle of an issue:
 
 ### The documentation
 
-We use [GitBook](https://www.gitbook.com/) for the [language documentation](https://crystal-lang.org/docs/).
+We use [MkDocs](https://www.mkdocs.org/) for [the language reference](https://crystal-lang.org/reference/).
```

Comment:
Could we just remove the reference to the doc generator? I don't think it adds any value here.

Links to https://crystal-lang.org/reference/ and https://github.com/crystal-lang/crystal-book for contribution should be enough. The details on how to contribute are then described there.

Reply:
Done. Not sure about wording